python/tests/plugins/first_party/test_primitive_validation.py

```diff
@@ -0,0 +1,78 @@
+from aac.lang.active_context_lifecycle_manager import get_active_context
+
+from aac.validate import validated_source
+from aac.io.constants import DEFINITION_SEPARATOR
+from aac.plugins.first_party.primitive_type_check.validators import int_validator
+
+from tests.active_context_test_case import ActiveContextTestCase
+from tests.helpers.assertion import assert_validator_result_success
+from tests.helpers.io import temporary_test_file
+from tests.helpers.parsed_definitions import create_definition
+from tests.helpers.prebuilt_definition_constants import (
+    TEST_TYPES_SCHEMA_DEFINITION,
+    TEST_TYPES_SCHEMA_EXTENSION_DEFINITION,
+    TEST_TYPES_VALID_INSTANCE,
+    TEST_TYPES_INVALID_INSTANCE,
+    TEST_TYPES_ROOT_KEY,
+    SCHEMA_FIELD_INT
+)
+
+
+class TestPrimitiveValidation(ActiveContextTestCase):
+
+    INTEGER_VALIDATOR = int_validator.get_validator()
+
+    VALID_PRIMITIVES_FILE_CONTENT = DEFINITION_SEPARATOR.join([
+        TEST_TYPES_SCHEMA_DEFINITION.to_yaml(),
+        TEST_TYPES_SCHEMA_EXTENSION_DEFINITION.to_yaml(),
+        TEST_TYPES_VALID_INSTANCE.to_yaml()
+    ])
+
+    INVALID_PRIMITIVES_FILE_CONTENT = DEFINITION_SEPARATOR.join([
+        TEST_TYPES_SCHEMA_DEFINITION.to_yaml(),
+        TEST_TYPES_SCHEMA_EXTENSION_DEFINITION.to_yaml(),
+        TEST_TYPES_INVALID_INSTANCE.to_yaml()
+    ])
+
+    def test_type_check_valid(self):
+        with temporary_test_file(self.VALID_PRIMITIVES_FILE_CONTENT) as test_file:
+            with validated_source(test_file.name) as result:
+                assert_validator_result_success(result)
+
+    def test_type_check_invalid(self):
+        with temporary_test_file(self.INVALID_PRIMITIVES_FILE_CONTENT) as test_file:
+            with self.assertRaises(Exception) as context:
+                with validated_source(test_file.name) as result: # noqa F841
+                    pass
+
+                self.assertIsNotNone(context.exception)
+
+                self.assertIn(self.INTEGER_VALIDATOR.name, context.exception)
+                self.assertIn(self.INTEGER_VALIDATOR.primitive_type, context.exception)
```

Comment:
I'm not 100% sure on this one but I think it should also work, here. That said, we can remove the Flake8 ignore by not creating a variable for the `validated_source` call.

```suggestion
        with (
            temporary_test_file(self.INVALID_PRIMITIVES_FILE_CONTENT) as test_file,
            self.assertRaises(Exception) as context,
            validated_source(test_file.name),
        ):
            self.assertIsNotNone(context.exception)

            self.assertIn(self.INTEGER_VALIDATOR.name, context.exception)
            self.assertIn(self.INTEGER_VALIDATOR.primitive_type, context.exception)
```